policyengine_us/variables/gov/states/id/tax/income/deductions/retirement_benefits/id_retirement_benefits_deduction.py

```diff
@@ -0,0 +1,37 @@
+from policyengine_us.model_api import *
+
+
+class id_retirement_benefits_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Idaho retirement benefits deduction"
+    unit = USD
+    documentation = "https://legislature.idaho.gov/statutesrules/idstat/title63/t63ch30/sect63-3022a/"
+    definition_period = YEAR
+    defined_for = StateCode.ID
```

Comment:
defined_for = "id_retirement_benefits_eligible_person"

Reply:
has to be at the same entity level, so this would only work if this were person-level. could this be reoriented toward intermediate person-level variables to make that work?